pkg/projects/Microsoft.NETCore.DotNetHost/Microsoft.NETCore.DotNetAppHost.builds

```diff
@@ -75,6 +79,10 @@
       <OSGroup>Linux</OSGroup>
       <Platform>amd64</Platform>
     </Project>
+    <Project Condition="'$(TargetsLinux)' == 'true' and '$(DistroRid)' == 'tizen.4.0.0-armel'" Include="tizen.4.0.0/Microsoft.NETCore.DotNetAppHost.pkgproj">
```

Comment:
We can't use debian armel. Because tizen uses differenct version of the libraries so it should be distributed with a different filename than debian armel. And the runtime id is also different. Our main goal is to get people to download .NET Runtime from .NET Core official websites. So we want to create seperate nupkg and tarball for tizen.